modular_newhorizons/master_files/code/modules/background/origins/origins/human/coalition.dm

```diff
@@ -0,0 +1,110 @@
+	important_information = "Because of Gadpathur's insular nature that is unwelcoming to outsiders and its hyper-militarized society, only native Gadpathurians may take the Gadpathuri accent. Many of the original colonists were from the indian subcontinent, thus many of the planet`s residents have their names or appearance tracing back to the founding roots of the colony."
+	possible_accents = list(ACCENT_GADPATHUR)
+	possible_citizenships = list(CITIZENSHIP_COALITION)
+	possible_religions = list(RELIGION_NONE, RELIGION_CHRISTIANITY, RELIGION_ISLAM, RELIGION_BUDDHISM, RELIGION_HINDU, RELIGION_TAOISM, RELIGION_JUDAISM, RELIGION_SIKHISM, RELIGION_OTHER)
+	origin_traits = list(TRAIT_ORIGIN_LIGHT_SENSITIVE)
+	origin_traits_descriptions = list("have a small resistance to radiation", "are more sensitive to bright lights")
+
+/singleton/origin_item/origin/gadpathur/on_apply(var/mob/living/carbon/human/H)
+  H.AddComponent(/datum/component/armor, list(rad = ARMOR_RAD_MINOR))
+
+/singleton/origin_item/origin/gadpathur_exile
+	name = "Gadpathurian Exile"
+	desc = "As an exile from Gadpathur you are, above all other things, a failure. A failure to meet the standards of Gadpathurian society found unworthy of defending it from the synthetic menace. You are a weakness that the planet and its people cannot afford to keep around, lest you compromise its defense. Stripped over everything that makes you Gadpathurian and cast aside to whoever will take you, it is now up to you -- and you alone -- to make something of yourself despite your status as a failure. The one certainty is that your home will not take you back, no matter what you do."
+	possible_accents = list(ACCENT_GADPATHUR)
+	possible_citizenships = list(CITIZENSHIP_COALITION, CITIZENSHIP_BIESEL, CITIZENSHIP_SOL)
+	possible_religions = RELIGIONS_COALITION
+	origin_traits = list(TRAIT_ORIGIN_LIGHT_SENSITIVE)
+	origin_traits_descriptions = list("have a small resistance to radiation", "are more sensitive to bright lights")
+
+/singleton/origin_item/origin/gadpathur_exile/on_apply(var/mob/living/carbon/human/H)
+  H.AddComponent(/datum/component/armor, list(rad = ARMOR_RAD_MINOR))
+
+/singleton/origin_item/origin/assunzione
+	name = "Republic of Assunzione"
+	desc = "One of the most remote planets colonized by humanity, the Republic of Assunzione is widely known for its lack of a functional sun - which burnt out mysteriously shortly after the planet was colonized - and its unusual native faith, Luceism. Luceism itself is an offshoot of traditional Abrahamic faiths and is centered around the worship of Ennoia, an abstract representation of light. Assunzionii society is quite insular and is centered around the planet's faith, with an overwhelming majority of the planet's residents adhering to Luceism."
```

Comment:
Replace with
> name = "Illuminated Cities of Assunzione"
> desc = "One of the most remote planets colonised by humanity, the Illuminated Cities of Assunzione are widely known for their lack of a functional sun - which burnt out mysteriously shortly after the planet was settled- and its unusual native faith, Luceism. Luceism itself is a melding of the faiths of its colonists with the prophecies of a small group of individuals, and is centered around the worship of Ennoia, an abstract representation of light. Assunzionii society is quite insular and is centered around the planet's faith, with an overwhelming majority of the planet's residents adhering to Luceism."
Also scrap the important_information section, we don't need it for this one.

Reply:
And done.
Would delete important info overall but since we are using overrides the original would still be there, so hopefully the commented out // works in disabling it.